app/src/main/java/grinnell/appdev/edu/lyles/menuItems/MenuItem.java

```diff
@@ -4,40 +4,18 @@
  * Created by mattmurphy on 9/25/16.
  */
 
-public class MenuItem {
+public interface MenuItem {
```

Comment:
`MenuItem` is a class in the Android SDK. You probably want to call this something else. Good that you're using an Interface!